test/testSXSModes.py

```diff
@@ -0,0 +1,373 @@
+                Mode numbers
+
+    Returns
+    -------
+    waveforms : dict
+                The waveform polarizations
+    """
+
+    ####################
+    # Prepare modes
+    ####################
+
+    errs = {}
+
+    aerrs = []
+    perrs = []
+
+    wf1_tlm = sxsw.get_mode(ell, emm)[:, 0] - mtime
+
+    if not (wf1_tlm - taxis1 == 0).all():
+        message('Difference in axis', wf1_tlm - taxis1)
+        raise ValueError("Time axis is different across modes!")
+
+    # nrcat
+    wf1_plm = sxsw.get_mode(ell, emm)[:, 1]
+    wf1_xlm = sxsw.get_mode(ell, emm)[:, 2]
+
+    wf1_lm = wf1_plm + 1j * wf1_xlm
+    wf2_lm = wf2.mode(ell, emm)
+
+    # Interpolate in amp, phase
+    wf1_Alm, wf1_Plm = xtract_camp_phase(wf1_lm.real, wf1_lm.imag)
+    wf2_Alm, wf2_Plm = xtract_camp_phase(wf2_lm.real, wf2_lm.imag)
+
+    wf1_rAlm = interp_resam_wfs(wf1_Alm, taxis1, taxis, resam_kind="cubic")
+    wf2_rAlm = interp_resam_wfs(wf2_Alm, taxis2, taxis, resam_kind="cubic")
+
+    wf1_rPlm = interp_resam_wfs(wf1_Plm, taxis1, taxis, resam_kind="cubic")
+    wf2_rPlm = interp_resam_wfs(wf2_Plm, taxis2, taxis, resam_kind="cubic")
+
+    # Get max locs
+    # maxloc1 = np.argmax(wf1_rAlm)
+    # maxloc2 = np.argmax(wf2_rAlm)
+
+    # if maxloc1!=maxloc2:
+    # message('Time axes are not centered! Recentering...')
+    # from waveformtools.waveformtools import roll
+    # wf1_rAlm = roll(wf1_rAlm, maxloc2-maxloc1)
+    # wf1_rPlm = roll(wf1_rPlm, maxloc2-maxloc1)
+    # raise ValueError('Time axes are not centered!')
+
+    # Ensure increasing phase
+    if np.mean(np.diff(wf1_rPlm)) < 0:
+        wf1_rPlm = -wf1_rPlm
+
+    if np.mean(np.diff(wf2_rPlm)) < 0:
+        wf2_rPlm = -wf2_rPlm
+
+    # return the waveforms
+    wf1_rlm = wf1_rAlm * np.exp(1j * wf1_rPlm)
+    wf2_rlm = wf2_rAlm * np.exp(1j * wf2_rPlm)
+
+    wf1_rlm = wf1_rlm / np.linalg.norm(wf1_rlm)
+    wf2_rlm = wf2_rlm / np.linalg.norm(wf2_rlm)
+
+    wf1_rlm_p = TimeSeries(wf1_rlm.real, delta_t=delta_t)
+    wf1_rlm_x = TimeSeries(wf1_rlm.imag, delta_t=delta_t)
+    wf2_rlm_p = TimeSeries(wf2_rlm.real, delta_t=delta_t)
+    wf2_rlm_x = TimeSeries(wf2_rlm.imag, delta_t=delta_t)
+
+    # dphase = wf1_rPlm - wf2_rPlm
+
+    # aerrs.append(res_amp)
+    # perrs.append(res_phase)
+
+    # errs.update({f'l{ell}m{emm}' : [res_amp, res_phase]})
+
+    # fig, ax = plt.subplots()
+    # ax.set_yscale('log')
+
+    # ax.plot(taxis, wf1_rlm_p, label='nrcat')
+    # ax.plot(taxis, wf2_rlm_p, label='wftools', linestyle='--')
+    # ax.set_title(f'waveforms l{ell}m{emm}')
+    # plt.grid()
+    # plt.show()
+
+    # fig, ax = plt.subplots()
+    # ax.set_yscale('log')
+
+    # ax.plot(taxis, np.absolute(dAmp_frac))
+    # ax.set_title(f'Amp diff l{ell}m{emm}')
+    # plt.grid()
+    # plt.show()
+
+    # fig, ax = plt.subplots()
+    # ax.set_yscale('log')
+    # ax.plot(taxis, np.absolute(dphase))
+    # ax.set_title(f'Phase diff l{ell}m{emm}')
+    # plt.grid()
+    # plt.show()
+
```

Comment:
If all these commented out lines are not needed, remove them pls. But if they are, perhaps let them be behind a boolean flag that can be switched?